sig-network/README.md

```diff
@@ -121,7 +121,7 @@ The following [subprojects][subproject-definition] are owned by sig-network:
   - [kubernetes/dns](https://github.com/kubernetes/dns/blob/master/OWNERS)
 ### network-policy
 - **Owners:**
-  - [kubernetes-sigs/kube-network-policies](https://github.com/kubernetes-sigs/kube-network-policies/blob/master/OWNERS)
+  - [kubernetes-sigs/kube-network-policies](https://github.com/kubernetes-sigs/kube-network-policies/blob/main/OWNERS)
```

Comment:
This is generated from https://github.com/kubernetes/community/blob/7f08c5694852747a46109abbf121ebccc7d11302/sigs.yaml#L2296-L2297.

So you should change the sigs.yaml and run `make`.